### ISSUE TITLE
My Steam App launches SADX in the Steam Version instead of the modded 2004 version.

### ISSUE THREAD

**Issue by FerdiNah** [X-Hax/sadx-mod-loader#146]
**Description**
Whenever I launch Sonic Adventure DX, It always runs the Steam Version, instead of the modded PC 2004 version I already have installed in the same folder.

**Steps Taken**
What I did was:
1. Go to settings
2. Then click on Properties
3. And verify integrity of game files.
4. Then it automatically restored and updated its files to the Steam Version (which i knew was going to happen and did on purpose just to experiment with the Steam version of the game)
5. Later, I tried opening the game through Steam assuming the PC 2004 installed version would run instead of the Steam version, but it did not.

**Expected behavior**
I expected SA Mod Loader to make it so Steam would run the modded version instead of the Steam version, like it did on the Legacy Mod Loader "SADX Mod Loader" whenever it detected the Steam Version being installed.

**Comment by cheatfreak47** [comment]
This isn't a bug, this is user error. "verify integrity of game files" in Steam will remove mods, since mods violate the integrity of the game files compared to what Steam expects to be there.

**Comment by FerdiNah** [comment]
Thank you for responding. Is there anything i can do to get Steam to launch the modded version of the game?

**Comment by FerdiNah** [comment]
And i'm sorry for putting the wrong label. I'm new to github and i didn't know too much about labels.

**Comment by cheatfreak47** [comment]
You'll have to rerun SADX Mod Installer or BetterSADX or whatever you did originally to convert the steam port. This question is the tech equivalent of dropping your ice cream cone on the sand at the beach and then asking how to get the sand off.

**Comment by FerdiNah** [comment]
Haha, that's a funny way to view it. Nevertheless, thanks for helping out!

**Comment by FerdiNah** [comment]
Tried with the SADX Mod Loader. Nothing worked. Now i'm gonna see if the BetterSADX application converts it. I don't want to risk losing my mods either, so I wonder if installing it will cause any harm to my precious collection of 124 mods installed.

**Comment by FerdiNah** [comment]
Whenever i try to delete the "Sonic Adventure  DX" file or the "AppLauncher", it always updates itself again to the Steam version. A mess. Shouldn't have caused such trouble when i clicked the "Verify integrity of files" option.

**Comment by cheatfreak47** [comment]
Firstly, I would recommend closing this issue, since this has nothing to do with SADX Mod Loader, and is entirely down to you destroying your own install of the game.

Second, my suggestion is to back up your "mods" folder and then delete the game entirely and completely reinstall the game clean from Steam and then run the SADX Mod Installer, and then copy your mods folder back over afterwards.

**Comment by FerdiNah** [comment]
OK! after a while, I got it to work. I had made a backup of the mods, then, replaced the mods folder with the one I had so my mods are back. I can launch the game through Steam, but not the SA Mod Manager, because when i click configuration it tries to run SADX Mod Manager but i removed all remains of SADX Mod Manager. It's perfectly fine with me, though. Anyways, thanks again for helping me with this little accident!
